test/forked/COWShed.t.sol

```diff
@@ -85,6 +85,22 @@ contract ForkedCOWShedTest is BaseForkedTest {
         userProxy.executeHooks(calls, nonce, _deadline(), signature);
     }
 
+    function testExecuteHooks_revertsOnInvalidEcdsaSignature() external {
+        Call[] memory calls = new Call[](0);
+        bytes32 nonce = "1337";
+        uint256 deadline = _deadline();
+        bytes memory signature = _signForProxy(calls, nonce, deadline, user);
+
+        // Invalidate signature.
+        unchecked {
```

Comment:
Instead of `unchecked`, would it work with

`signature[0] = bytes1((uint8(signature[0]) + 1) % 256);` 
or 
`signature[0] = bytes1(uint8(signature[0]) ^ 0x01)`